lang/src/org/partiql/lang/eval/EvaluatingCompiler.kt

```diff
@@ -889,6 +889,49 @@ internal class EvaluatingCompiler(
         }
     }
 
+    private fun compileLimit(limitClause: ExprNode, env: Environment) : Long {
```

Comment:
This was mostly moved from [`compileQueryWithoutProjection `](https://github.com/partiql/partiql-lang-kotlin/blob/master/lang/src/org/partiql/lang/eval/EvaluatingCompiler.kt#L1365).